airbyte-cdk/bulk/toolkits/load-object-storage/src/main/kotlin/io/airbyte/cdk/load/command/object_storage/ObjectStorageFormatSpecification.kt

```diff
@@ -180,36 +183,36 @@ class ParquetFormatSpecification(
 
     @JsonSchemaTitle("Compression Codec")
     @JsonPropertyDescription("The compression algorithm used to compress data pages.")
-    @JsonProperty("compression_codec")
+    @JsonProperty("compression_codec", defaultValue = "UNCOMPRESSED")
```

Comment:
Is `UNCOMPRESSED` different than `No Compression`? Maybe it's a parquet thing?

Reply:
Yes. The codecs used by avro and parquet compress data internal to the file, and are enums specific to the libraries we use.

The compression used by json and csv is applied after-the-fact to the whole file, and is just gzip.